README.md

```diff
@@ -66,12 +66,11 @@ Selenium docker is provided as a convenience to make selenium testing easier and
 
 Under the key `seleniumDocker` in the wdio.conf.js you can pass a configuration object with the following structure:
 
+* **composeFile** - The docker compose file to use to standup the hub. Defaults to a compose file with 5 chrome instances on port http://localhost:4444/wd/hub.
 * **enabled** - Flag to disable selenium docker; useful for CI environments which can startup the docker instance outside of test runs. Defaults to true.
-* **cleanup** - Destroy the docker container after the test run. Defaults to false.
```

Comment:
removed this option because docker highly recommends removing containers after every test run